src/Spryker/Client/ProductLabelStorage/Storage/Dictionary/LabelDictionary.php

```diff
@@ -102,14 +105,15 @@ protected function initializeLabelDictionary($localeName)
 
     /**
      * @param string $localeName
+     * @param string|null $storeName
      *
      * @return \Generated\Shared\Transfer\ProductLabelDictionaryStorageTransfer
      */
-    protected function readLabelDictionary($localeName)
+    protected function readLabelDictionary($localeName, ?string $storeName = null)
```

Comment:
Missed type hint for `$localName` previously it was `string`.

Reply:
emm, actually not, please, re-check